test/testleakautovar.cpp

```diff
@@ -760,6 +761,22 @@ class TestLeakAutoVar : public TestFixture {
         ASSERT_EQUALS("[test.c:4]: (error) Resource leak: q\n", errout_str());
     }
 
+    void fdopen() { // #12781
```

Comment:
I wonder if this test is effective since posix.cfg doesn't seem to be loaded.

Reply:
Fix: https://github.com/danmar/cppcheck/pull/6980